theano/__init__.py

```diff
@@ -29,6 +29,12 @@
 # theano code, since this code may want to log some messages.
 import logging
 
+import sys
+
+if sys.platform == 'win32' and sys.version_info[0:2] == (3, 5):
+    raise RuntimeError(
+        "Theano do not support Python 3.5 on Windows. Use Python 2.7 or 3.4.")
```

Comment:
Since python 3.5 exist as it change all the way the c linking is done on windows. See gh-3376